demo/widgets/index.html

```diff
@@ -229,6 +257,32 @@
     </table>
   </div>
 </template>
-</dom-bind>
+<script>
+  (function() {
+    'use strict';
+
+    Polymer({
+      is: 'nuxeo-widget-demo',
+      properties: {
+        directoryCheckboxItems: Array
+      },
+
+      _formatDirectotryItems: function(items) {
```

Comment:
Typo

Reply:
Also, I'd rather output unformatted values than having to wrap the demo in an element....